packages/flame/lib/src/effects/combined_effect.dart

```diff
@@ -1,109 +1,70 @@
 import 'dart:math';
 import 'dart:ui';
 
-import '../components/position_component.dart';
+import '../../components.dart';
 import 'effects.dart';
 
-class CombinedEffect extends PositionComponentEffect {
-  final List<PositionComponentEffect> effects;
-  final double offset;
+/// The [CombinedEffect] is just a container for multiple effects, all settings
+/// are handled on each individual effect. [CombinedEffect] is not an effect
+/// itself.
+class CombinedEffect extends ComponentEffect with EffectsHelper {
```

Comment:
What is this `EffectsHelper`? I am not finding it on the PR and seems that it is a new thing right?

Reply:
```dart
mixin EffectsHelper on Component {
  void clearEffects() {
    children.removeAll(effects);
  }

  List<ComponentEffect> get effects {
    return children.isRegistered<ComponentEffect>()
        ? children.query<ComponentEffect>()
        : [];
  }
}
```
For now it is just those two helper methods (it lives in `effects.dart`).